app/lib/katello/resources/cdn.rb

```diff
@@ -1,7 +1,12 @@
 module Katello
   module Resources
     module CDN
-      SUPPORTED_SSL_VERSIONS = ['SSLv23', 'TLSv1'].freeze
+      SUPPORTED_TLS_VERSIONS = [
+        'TLSv1',
+        'TLSv1.1',
+        'TLSv1.2',
+        *('TLSv1.3' if defined?(OpenSSL::SSL::TLS_1_3_VERSION))
```

Comment:
Shouldn't be there `OpenSSL::SSL::TLS1_3_VERSION` ? (`s/TLS_1/TLS1/`)?

Reply:
Yes! Good catch, thanks